torch/quantization/fx/utils.py

```diff
@@ -469,3 +469,15 @@ def is_get_tensor_info_node(node: Node) -> bool:
     result: bool = \
         node.op == "call_function" and node.target == getattr and node.args[1] == "shape"  # type: ignore[assignment]
     return result
+
+def get_next_node(model: GraphModule, node: Node) -> Optional[Node]:
```

Comment:
Can we just do `node.next()`?

Reply:
I tried using `node.next` but it didn't give the wanted values. For example, for the graph with 2 consecutive linear layers, if node is `linear`, `get_next_node()` would return `linear_1` which we can use to search in `matches`, but `node.next` would return `linear2_w` which doesn't exist in `matches`.

Reply:
Would it work if we kept calling node.next in a loop until we hit a `('call_module', 'call_method', 'call_function', 'output')` type?